### ISSUE TITLE
Bug - Not correctly documenting collecting using 'collect' objective

### ISSUE THREAD

**Issue by ghost** [CitizensDev/Citizens#678]
Citizens ver: Latest stable (1.1.4)
Bukkit ver: Latest recommended (#1818)

Here is the quest.yml file:

http://pastebin.com/G4WxKWtG

So I accept the quest and go out to find wood. I find a tree, axe a piece of wood, and collect it. I check /quest status and it says that I have collected 1 Log, 1 Coal, and 1 Iron. I collect another wood, and it now says I have collected 2 Log, 2 Coal, and 2 Iron.

I also tested collecting Iron and Coal and the same issue persisted. Assuming there is some kind of bug updating the quest status.

**Comment by aufdemrand** [comment]
Verified issue, same setup.  Perhaps someone still running 1.1.3 can test?

**Comment by FePython** [comment]
yeah i can also just toss any item to the ground and it will count.

lets say the quest is to get 10 coal, i toss 10 sticks from my inventory to the ground and pick it back up and it says that i collected 10 coal

useing 1.1.5
